common/src/test/java/io/netty/util/concurrent/FastThreadLocalTest.java

```diff
@@ -238,4 +241,52 @@ protected void onRemoval(String value) throws Exception {
             onRemovalCalled.set(value);
         }
     }
+
+    @Test
+    public void testConstructionWithIndex() throws Exception {
```

Comment:
I wonder if we should disable this test by default as it will use a lot of memory. WDYT ?

Reply:
I think the GC will kicked in and recycle the memory while in the 'loop', so it will not use a lot of memory.